assignments/assignment7/resume.scss

```diff
@@ -0,0 +1,377 @@
+    border-radius: 50%;
+    border: 4px $second-color solid;  
+ }
+  .contacts{
+    font-size: 15px;
+    width: 100%;
+    overflow: hidden;
+  }
+ 
+ .header{
+    width: 100%;
+    margin-bottom: 10px;
+    padding: 1em;
+    margin: 0 -1em;
+    border-bottom: 2px $second-color solid;
+    overflow: hidden;
+    
+}
+
+.details{
+    width: 100%;
+    overflow: hidden;
+}
+.detailsLeft{
+    float: left;
+    width: 50%;
+    overflow: hidden;
+    
+}
+.detailsRight{
+    float: right;
+    width: 50%;
+    overflow: hidden;
+}
+body{
+    padding: 30px 20px 10px;
+}
+h2{
+    font-size: 25px;
+    margin-bottom: 10px;
+}
+h1{
+    font-weight: lighter; 
+    font-size: 60px;
+    margin-bottom: 10px;
+    display: inline-block;
+    line-height: 50px;
+}
+h4{
+    margin-bottom: 10px;
+}
+
+.contacts {
+    i{
+        float: right;
+        width: 7%;
+    }
+    p{
+        float: left;
+        width: 93%;
+    }
+    address{
+        display: initial;
+        overflow: hidden;
+        width: 95%;
+    }
+ }
+
+.shortDesc{
+    font-size: 18px;
+}
+.sections{
+    text-decoration: none;
+    margin-bottom: 20px;
+    margin-top: 10px;
+}
+.when{
+    font-style: italic;
+    font-size: 15px;
+    float: left
+}
+.where{
+    font-style: italic;
+    font-size: 15px;
+    float: right;
+    margin-right: 20%;  
+}
+
+.subSections{
+   margin-top: 5px;
+   margin-bottom: 5px;
+   clear: both;
+}
+ul{
+    margin-bottom: 20px;
+    list-style: none;   
+}
+li:before{
+    content: '\2022';
+    color: $second-color;
+}
+
+ .languages , .achievments, .progressList {
+     li:before{
+         content: none;
+         list-style-type: none;
+     }
+    
+}
+.skillsAndCompetences{
+    margin-bottom: 10px;
+}
+.education li {
+    display: inline-block;
+    width: 40%;
+    margin-right: 30px;
+    list-style-position: inside;
+    text-indent: -0.5em;
+    padding-left: 0.5em;
+    box-sizing: border-box;
+}
+.languages li{
+   display: inline-block;
+   width: 40%;
+   margin-bottom: 0;
+}
+.languages li:nth-child(n+3){
+    font-style: italic;
+    color: gray;
+}
+li{ 
+     margin-bottom: 10px;
+}
+.interests{
+font-size: 17px;
+}
+.companyName{
+    font-size: 20px;
+    margin-bottom: 10px;
+}
+a{
+    font-size: 15px;
+}
+.fa-envelope, .fa-phone, .fa-linkedin, .fa-github{
+    color: $second-color;
+}
+.skills{
+    display: inline-block;
+    width: 30%;
+    font-size: 17px;
+    height: 100%;
+    margin-bottom: 15px;
+    margin-top: 20px;
+}
+.scaleBar{
+    position: absolute;
+    width: 1px;
+    height: 9px;
+    background-color: darkslategray;
+}
+#degree1{
+    @extend .scaleBar;
+    right: 16%;
+}
+#degree2{
+     @extend .scaleBar;
+    right: 22%;
+}
+#degree3{
+    @extend .scaleBar;
+    right: 29%;
+}
+#degree4{
+    @extend .scaleBar;
+    right: 35%;
+}
+
+.leftColoredRect{
+  position: absolute;
+  left: 0;
+  width: 20px;
+  height: 90px;
+  background-color: darkslategray;
+}
+#rect1{
```

Comment:
You don't need id selector for each rectangle. Just use one class to target all of them.